src/rosdiscover/ros2_translator/arch.py

```diff
@@ -0,0 +1,41 @@
+import yaml
```

Comment:
I'm not 100% sure about the YAML implementation/package that's used by ROS2, but the `yaml` module isn't part of the Python standard library, so there's always a risk here that we will try to use code that isn't available inside the Docker container for a particular robot.

A more portable alternative here would be to use the `json` module, which is included in the Python standard library. Using `json` should have minimal impact on the rest of the script.